misc/identity-keys/CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # @walletconnect/identity-keys
 
+## 1.0.0
+
+### Major Changes
+
+- Refactor registration to be multi function
```

Comment:
Could we note what has actually changed in the public API pls? 🙏 

E.g. `registerIdentity` has been removed in favour of `prepareRegistration` + ...